src/workflows-app/components/WorkflowCard.ts

```diff
@@ -48,6 +51,15 @@ export type WorkflowCardProps = {
 
 export const WorkflowCard = ({ method, subCard, children }: PropsWithChildren<WorkflowCardProps>) => {
   const isWorkflowSet = 'methods' in method; // Used to narrow type
+  let isPrivate;
+  if (isWorkflowSet || method.isPrivate == null) {
```

Comment:
TOL: it might be good to add the case for `method.isPrivate` being `undefined` since until CBAS is upgraded it won't contain `isPrivate` in its API response. What do you think?

Reply:
Sounds good. Thank you!